lib/src/allocation.dart

```diff
@@ -25,39 +25,31 @@ typedef PosixFree = void Function(Pointer);
 final PosixFree posixFree =
     stdlib.lookupFunction<PosixFreeNative, PosixFree>('free');
 
-typedef WinGetProcessHeapFn = Pointer Function();
-final WinGetProcessHeapFn winGetProcessHeap = stdlib
-    .lookupFunction<WinGetProcessHeapFn, WinGetProcessHeapFn>('GetProcessHeap');
-final Pointer processHeap = winGetProcessHeap();
+typedef WinCoTaskMemAllocNative = Pointer Function(Size cb);
+typedef WinCoTaskMemAlloc = Pointer Function(int cb);
+final WinCoTaskMemAlloc winCoTaskMemAlloc =
+    stdlib.lookupFunction<WinCoTaskMemAllocNative, WinCoTaskMemAlloc>(
+        'CoTaskMemAlloc');
 
-typedef WinHeapAllocNative = Pointer Function(Pointer, Uint32, IntPtr);
-typedef WinHeapAlloc = Pointer Function(Pointer, int, int);
-final WinHeapAlloc winHeapAlloc =
-    stdlib.lookupFunction<WinHeapAllocNative, WinHeapAlloc>('HeapAlloc');
-
-typedef WinHeapFreeNative = Int32 Function(
-    Pointer heap, Uint32 flags, Pointer memory);
-typedef WinHeapFree = int Function(Pointer heap, int flags, Pointer memory);
-final WinHeapFree winHeapFree =
-    stdlib.lookupFunction<WinHeapFreeNative, WinHeapFree>('HeapFree');
-
-// ignore: constant_identifier_names
-const int HEAP_ZERO_MEMORY = 8;
+typedef WinCoTaskMemFreeNative = Void Function(Pointer pv);
+typedef WinCoTaskMemFree = void Function(Pointer pv);
+final WinCoTaskMemFree winCoTaskMemFree = stdlib
+    .lookupFunction<WinCoTaskMemFreeNative, WinCoTaskMemFree>('CoTaskMemFree');
 
 /// Manages memory on the native heap.
 ///
 /// Does not initialize newly allocated memory to zero. Use [_CallocAllocator]
 /// for zero-initialized memory on allocation.
 ///
 /// For POSIX-based systems, this uses `malloc` and `free`. On Windows, it uses
-/// `HeapAlloc` and `HeapFree` against the default public heap.
+/// `CoTaskMemAlloc`.
```

Comment:
Now that we change the underlying implementation, we break people if they were mixing and matching `package:ffi` and their own calls to `HeapAlloc` and `HeapFree`.

Let's bump `package:ffi` to 2.0.0.